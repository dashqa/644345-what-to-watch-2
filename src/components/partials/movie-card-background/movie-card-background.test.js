import React from 'react';
import renderer from 'react-test-renderer';
import MovieCardBackground from './movie-card-background';

const props = {
  name: `Title`,
  backgroundImage: `http://image.com/image.png`,
};

it(`Movie card background correctly renders after relaunch`, () => {
  const tree = renderer.create(<MovieCardBackground {...props}/>).toJSON();
  expect(tree).toMatchSnapshot();
});
