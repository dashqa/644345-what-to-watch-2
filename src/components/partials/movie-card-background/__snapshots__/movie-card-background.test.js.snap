// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movie card background correctly renders after relaunch 1`] = `
<div
  className="movie-card__bg"
>
  <img
    alt="Title"
    src="http://image.com/image.png"
  />
</div>
`;
